relational_operators/TextScanOperator.cpp

```diff
@@ -19,6 +19,12 @@
 
 #include "relational_operators/TextScanOperator.hpp"
 
+#include "relational_operators/RelationalOperatorsConfig.h"  // For QUICKSTEP_HAVE_UNISTD.
```

Comment:
Can you move this include between line 39 and 50, i.e. in the block of included files from quickstep modules. When you do that, make sure the list is sorted.